packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5489,17 +5489,36 @@ export default class Meeting extends StatelessWebexPlugin {
           },
         };
 
+        const streamsToUnpublish = [];
+
+        if (this.mediaProperties?.audioStream) {
+          streamsToUnpublish.push(this.mediaProperties?.audioStream);
+        }
+
+        if (this.mediaProperties?.videoStream) {
+          streamsToUnpublish.push(this.mediaProperties?.videoStream);
+        }
+
+        if (this.mediaProperties?.shareAudioStream) {
+          streamsToUnpublish.push(this.mediaProperties?.shareAudioStream);
+        }
+
+        if (this.mediaProperties?.shareVideoStream) {
+          streamsToUnpublish.push(this.mediaProperties?.shareVideoStream);
+        }
+
+        if (streamsToUnpublish.length) {
+          this.unpublishStreams(streamsToUnpublish);
```

Comment:
unpublishStreams() has a check for each member of the passed in array to be truthy and skips the ones that aren't, so instead of the block of lines 5494-5508 you could just call it like this:
```
this.unpublishStreams([
  this.mediaProperties.audioStream,
  this.mediaProperties.videoStream,
  this.mediaProperties.shareAudioStream,
  this.mediaProperties.shareVideoStream
]
```
if any of the streams are undefined it will skip those and only unpublish the ones that are defined. Also `this.mediaProperties` is guaranteed to exist, so no need for the optional chaining operator.

Reply:
Took care of this comment.